pwa/src/components/logs/requestTable.tsx

```diff
@@ -3,7 +3,7 @@ import { Table, Modal, Spinner, Card, Alert, Tabs, Accordion } from "@conduction
 import { isLoggedIn } from "../../services/auth";
 import FlashMessage from 'react-flash-message';
 
-export default function RequestTable({ id = null }) {
+export default function RequestTable({ entityId = null }) {
```

Comment:
I don't feel as if this entire file should be updated (would be out of scope for this pull request). However, I think the following changes could be made:

1. Typing of the component it self, so: `export const RequestTable: React.FC<RequestTableProps> = () => { ... }`
2. Addition of the `RequestTableProps` interface, with an `entityId` prop;
3. Remove the setter (`entityId = null`), set the `entityId` when implementing the component, not within the component;

Don't forget to add `export default RequestTable` at the bottom of component.

Reply:
de response en request table komen te vervallen en daarvoor in de plaats willen we de LogTable gebruiken. Dus die pas ik aan en verwijder dan de ongebruikte files

Reply:
Perfect! Even better.